docs/package.json

```diff
@@ -131,7 +130,6 @@
     "@types/react-swipeable-views-utils": "^0.13.4",
     "@types/react-transition-group": "^4.4.6",
     "@types/react-window": "^1.8.5",
-    "@types/styled-components": "5.1.26",
```

Comment:
`styled-components` v6 is written in TypeScript, no need for `@types/styled-components` anymore.